changecalculator/src/main/java/org/msstate/cse/changecalculator/main/Main.java

```diff
@@ -0,0 +1,111 @@
+package org.msstate.cse.changecalculator.main;
+

```

Comment:
There isn't a header that adequately explains how the program works. Also, the comments are not detailed.